### DISCUSSION TITLE
💖 Help Support Signs

### DISCUSSION THREAD

**Discussion by ryanlua** [ryanlua/Signs#26] [Announcements]
# Help Support the Development of Signs

> [If you are enjoying Signs, please purchase it on the Creator Marketplace.](https://create.roblox.com/marketplace/asset/6994955669)

**This is just a PA that you can support me and Signs by buying [Signs from the Creator Marketplace](https://create.roblox.com/marketplace/asset/6994955669) to help support its development.** Signs is completely open-source and free expect for Signs from the Creator Marketplace which is used to support us. I've put much of hours into the development of Signs to release its entire source for free.

**You can gain all the features of Signs for free without having to pay.** There we don't do paywalls or ads to say that you should buy Signs in the plugin. 

**Due to Signs open-source nature a lot of people will just get Signs for free.** I'm not shaming that and was at the point of using free software only too. I'm asking if you really like Signs or any of the other products that you can help support me and the development of aforementioned products by just sparing 24 cents worth of Robux. 

**They have been a lot of people thinking that Signs is a fully paid plugin, which is false.** The [Readme](https://github.com/RyanLua/Signs#readme) has been updated to better reflect that Signs can be purchased for free and relies on donations more or less for development.

Thanks for all your support for Signs with much love,
@RyanLua 

###### **Help support Signs from for 100 Robux or 24 cents here:** [Signs - Creator Marketplace](https://create.roblox.com/marketplace/asset/6994955669).

**Comment by ghost** [comment]
Too bad I bought the plugin and supported you
